app.py

```diff
@@ -120,6 +120,11 @@ def log_out():
 def libraries():
     return app.library_registry.registry_controller.libraries()
 
+@app.route('/admin/libraries_opds')
+@returns_problem_detail
+def libraries_opds():
+    return app.library_registry.registry_controller.libraries_opds()
+
```

Comment:
This is talking to SimplyE, not the admin interface, so we don't want `/admin/` in the route. Just routing it from `'/libraries` should be fine.